chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml

```diff
@@ -72,7 +72,8 @@ spec:
         args:
         - "--node-name=$(MY_NODE_NAME)"
         - "--csi-socket={{ default .Values.csi.node.pluginMountPath .Values.csi.node.pluginMounthPath }}/{{ .Values.csi.node.socketPath }}"
-        - "--grpc-endpoint=$(MY_POD_IP):50053"
+        - "--grpc-ip=$(MY_POD_IP)"
```

Comment:
not directly related to this change, but I'm not sure why template has grpc port 50053 here while code sets 50051 as default for csi-node? @tiagolobocastro @Abhinandan-Purkait

Reply:
I don't think there is any specific reason. We probably did this because core-agent also uses 50051 and thus to keep things unique. Although won't matter IMO. @dsharma-dc Did you foresee any issues?

Reply:
If core agent is using 50051 then I guess that's the reason.